src/scss/loginPage.scss

```diff
@@ -0,0 +1,109 @@
+$text-color: #947a74;
+$btn-color: $text-color;
+
+
+html,
+body {
+  margin: 0px;
+  height: 100%;
+}
+
+.page-wrapper {
+  height: 100%;
+  display: flex;
+}
+
+.login-wrapper {
+  height: 100%;
+  width: 50%;
+  display: flex;
+  justify-content: center;
+}
+
+.login-box {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  font-family: Helvetica;
+  color: $text-color;
+  width: 50%;
+}
+
+.login-subheader {
+  margin: 0px;
+  font-family: 'Dancing Script', cursive;
+  font-size: 30px;
+}
+
+.login-header {
+  text-align: center;
+  text-transform: uppercase;
+  margin: 0px;
+  font-family: 'Space Mono', monospace;
+}
+
+.login-form {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin-top: 60px;
+}
+
+.login-label {
+}
+
+.login-input {
+  border: none;
```

Comment:
Is it needed?

Reply:
Yes, by default there is a border which we want to disable in order to get only the bottom line.